curb.gemspec

```diff
@@ -1,7 +1,7 @@
 Gem::Specification.new do |s|
   s.name    = "curb"
   s.authors = ["Ross Bamford", "Todd A. Fisher"]
-  s.version = '0.9.10.skroutz1'
+  s.version = '0.9.11.skroutz1'
```

Comment:
Since the upstream hasn't released this as `0.9.11` (yet?), let's bump the skroutz part of the version string here instead, i.e. I would suggest to make this `0.9.10.skroutz2`.

Reply:
Updated [here](https://github.com/skroutz/curb/pull/1/commits/cdc554dcd925f4429cc238f3e36e218d91d926fe).